cfme/tests/containers/test_breadcrumbs.py

```diff
@@ -23,38 +24,53 @@
     pytest.mark.provider([ContainersProvider], scope='module')]
 
 
-DataSet = namedtuple('DataSet', ['obj', 'allView'])
+DataSet = namedtuple('DataSet', ['obj', 'collection_obj'])
 
 
 TESTED_OBJECTS = [
-    Service, Route, Project, Pod, Image, ContainersProvider, ImageRegistry,
-    Node, Replicator, Template, Volume
+    DataSet(Service, ServiceCollection),
+    DataSet(Route, RouteCollection),
+    DataSet(Project, ProjectCollection),
+    DataSet(Pod, PodCollection),
+    DataSet(Image, ImageCollection),
+    DataSet(ContainersProvider, None),
+    DataSet(ImageRegistry, ImageRegistryCollection),
+    DataSet(Node, NodeCollection),
+    DataSet(Replicator, ReplicatorCollection),
+    DataSet(Template, TemplateCollection),
+    DataSet(Volume, VolumeCollection)
 ]
 
 
+def clear_search(view):
```

Comment:
OPTIONAL: I would rather add this resetter

Reply:
This will not be applied in this case because the navigation is done via the breadcrumbs:
```python
view.breadcrumb.click_location(data_set.obj.all_view.SUMMARY_TEXT)
view = appliance.browser.create_view(data_set.obj.all_view)
clear_search(view)
```